beacon_node/lighthouse_network/src/rpc/handler.rs

```diff
@@ -892,6 +903,15 @@ where
             self.shutdown(None);
         }
 
+        // If we received a Ping, we queue a Pong response.
+        if let InboundRequest::Ping(ping) = req {
+            trace!(self.log, "Received Ping, queueing Pong";"connection_id" => %self.id, "peer_id" => %self.peer_id);
+            self.send_response(
+                self.current_inbound_substream_id,
+                RPCCodedResponse::Success(RPCResponse::Pong(ping)),
```

Comment:
Here, the request (the `ping` variable) is set to the response, but we should set our sequence number as `Network::pong()` was doing before this change, right?

https://github.com/sigp/lighthouse/blob/672dcbd868198ec6430c90f9d41373cde740dc3d/beacon_node/lighthouse_network/src/service/mod.rs#L1131-L1139

Reply:
Thanks Akihito, I definitely overlooked this, will submit a follow up PR bringing the sequence number to the `Behaviour` and probably we can reply the pings there. CC @AgeManning

Reply:
submitted https://github.com/sigp/lighthouse/pull/6408